### PR TITLE
Basic BGP Establish Test

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

File: bgp/bgp.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package main
```
Comment (wenovus) [inline bgp/bgp.go:1]: TODO: Delete this experimental file, it's not used by the test.

File: kne/twodut_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+topology: /usr/local/google/home/wenbli/gocode/src/github.com/openconfig/lemming/kne/twodut_topo.pb.txt
```
Comment (DanG100) [inline kne/twodut_config.yaml:1]: don't check in this file

File: kne/deploytwodut.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+cd "$(dirname "$0")"
+
+kne delete ./twodut_topo.pb.txt || true
+cd ..
+DOCKER_BUILDKIT=1 docker build . -t "lemming:latest"
+kind load docker-image lemming:latest --name kne
+cd -
+kne create ./twodut_topo.pb.txt
```
Comment (DanG100) [inline kne/deploytwodut.sh:10]: ideally modify deploy.sh to have a topo argument. i can fix that in follow up PR

File: dataplane/handlers/interface.go [hunk 4]
```diff
@@ -302,6 +302,8 @@ func (ni *Interface) handleAddrUpdate(ctx context.Context, au *netlink.AddrUpdat
 		}
 		if err := engine.AddLayer3PuntRule(ctx, ni.fwd, modelName, ipBytes); err != nil {
 			log.Warningf("failed to add layer3 punt rule: %v", err)
+		} else {
```
Comment (DanG100) [inline dataplane/handlers/interface.go:305]: remove this, i added logging in the other PR

**Review by DanG100 — APPROVED** [review]

File: kne/deploytwodut.sh (hunk 3 above)
Comment (wenovus) [reply kne/deploytwodut.sh:10]: It's a bit annoying because of the `dirname`, I'll let you decide what to do there.

**Pull request merged**